### PR TITLE
Add productArch to package

### PR DESCRIPTION
**Pull request by dmonroym**
This is in support of work for dev17

### CODE REVIEW

File: setup/Swix/Microsoft.NuGet.Build.Tasks.Setup/Microsoft.NuGet.Build.Tasks.Setup.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>	
     <TargetFramework>net461</TargetFramework>	
     <VisualStudioInsertionComponent>Microsoft.NuGet.Build.Tasks</VisualStudioInsertionComponent>
+	<SetupProductArch>neutral</SetupProductArch>
```
Comment (genlu) [inline setup/Swix/Microsoft.NuGet.Build.Tasks.Setup/Microsoft.NuGet.Build.Tasks.Setup.csproj:5]: You are using a swr file for this package, so the property needs to be added there instead. e.g.
https://github.com/genlu/roslyn/blob/901d793eb6c1531feb08a88f2434ba7834cec974/src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.Setup.csproj#L140

Comment (dmonroym) [reply setup/Swix/Microsoft.NuGet.Build.Tasks.Setup/Microsoft.NuGet.Build.Tasks.Setup.csproj:5]: Thanks! Previously attempted that but hadn't updated versions of packages so it was failing. I've done it correctly now on #107

**Pull request merged**